src/bgfx_shader.sh

```diff
@@ -601,16 +601,18 @@ vec4  mod(vec4  _a, vec4  _b) { return _a - _b * floor(_a / _b); }
 #	define ISAMPLER3D(_name, _reg) uniform isampler3D _name
 #	define USAMPLER3D(_name, _reg) uniform usampler3D _name
 
-#	define texture2DBias(_sampler, _coord, _bias)      texture2D(_sampler, _coord, _bias)
-#	define textureCubeBias(_sampler, _coord, _bias)    textureCube(_sampler, _coord, _bias)
-
-#	if BGFX_SHADER_LANGUAGE_GLSL >= 130
+#	if BGFX_SHADER_LANGUAGE_GLSL < 130
```

Comment:
This is wrong, because `BGFX_SHADER_LANGUAGE_GLSL` is 0 when it's not GLSL. Keep old `#if` logic, and move this part into `#else` block.